x-pack/plugins/lens/public/drag_drop/announcements.tsx

```diff
@@ -113,13 +187,29 @@ const defaultAnnouncements = {
         label,
       },
     }),
-  cancelled: () =>
-    i18n.translate('xpack.lens.dragDrop.announce.cancelled', {
-      defaultMessage: 'Movement cancelled',
-    }),
+  cancelled: ({ label, groupLabel, position }: HumanData) => {
```

Comment:
Something very strange is happening with the cancellation motion- I can't press `escape` any more in the following scenario:

1. Select a bucket in a reorderable group, like on data table, by pressing spacebar
2. Press escape
3. Nothing happens

Reply:
I checked in one- and multi- elements group (buckets for data table) with and without VoiceOver on and I am not able to reproduce it 😭

Reply:
I checked again and I cannot find any problems with it. I will merge this PR as I think even if you can reproduce it, it probably not caused by it. I'd appreciate an issue with some screenshots, if you still can reproduce it.